### ISSUE TITLE
Poll results are not shown 

### ISSUE THREAD

**Issue by BANOnotIT** [mastodon/mastodon-android#544]
https://pleroma.dark-alexandr.net/objects/0dda71ed-3d2a-4883-b7be-19f45a42855a
In this poll I only see my selected options while the results are not shown at all.
![Screenshot_2023-03-12-13-44-36-726_org.joinmastodon.android.sk.jpg](https://user-images.githubusercontent.com/11363215/224539625-5c87c21f-560d-461d-866d-d76d74fd78bd.jpg)

**Comment by grishka** [comment]
Is this a fork? I don't remember polls having square checkboxes and options being multi-line and borderless.

**Comment by BANOnotIT** [comment]
Yeah, this is a fork. But same thing replicates on official build from Google play v1.2.0 (50)
![Screenshot_2023-03-13-15-07-51-732_org.joinmastodon.android.jpg](https://user-images.githubusercontent.com/11363215/224697540-c9b33276-fcdd-4760-83d5-21fbbc6d95fb.jpg)

**Comment by grishka** [comment]
Which server are you on? Do the results display correctly on the web?

**Comment by BANOnotIT** [comment]
https://mastodon.ml/@fmn@pleroma.dark-alexandr.net/109987159785026944 yeah, it's correctly interpreted in the web interface.
